resources/xsd/DFDL_part3_model.xsd

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I assume all these files in `resources/xsd` have just been copied and unmodified from Daffodil? Can they instead be pulled out of the the daffodil-lib dependency?

Otherwise these files may start to drift from the ones in Daffodil when we make changes and we'll have to remember (and most likely forget) to keep things updated.

Reply:
Yes, that's where they came from. I will give that a shot.